mitodl/edx-api-client
# Replace Travis with Github actions

#### What are the relevant tickets?
Fixes #72 

#### What's this PR do?
Switches over to github actions. Travis is discontinuing free open source credits.

#### How should this be manually tested?
View the checks below the PR. Make sure all tests pass and that coverage is uploaded correctly.

#### Any background context you want to provide?
I decided to keep pytest-pylint and tox in the interest of time
